### PR TITLE
[BEAM-7765] - Add test for snippet accessing_valueprovider_info_after_run

### PR DESCRIPTION
**Pull request by angulartist**
Added unit test for snippet accessing_valueprovider_info_after_run

- Removed unused import (WriteToText)
- Updated bad function call

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by angulartist** [comment]
R: @pabloem

**Comment by pabloem** [comment]
Run Portable_Python PreCommit

File: sdks/python/apache_beam/examples/snippets/snippets.py [hunk 1]
```diff
@@ -1394,28 +1397,24 @@ def __init__(self, string_vp):
...
-  # [END AccessingValueProviderInfoAfterRunSnip1]
+  # Create pipeline.
+  with beam.Pipeline(options=my_options) as p:
+    # Add a branch for logging the ValueProvider value.
+    _ = (p
+         | beam.Create([None])
+         | 'LogValueProvider' >> beam.ParDo(
+              LogValueProvidersFn(my_options.string_value)))
+
+    # The main pipeline.
```
Comment (pabloem) [inline sdks/python/apache_beam/examples/snippets/snippets.py]: Does it make sense to completely move the snippet to the `snippets_test.py` file, and remove it from snippets.py?

File: sdks/python/apache_beam/examples/snippets/snippets_test.py [hunk 2]
```diff
@@ -1266,6 +1266,59 @@ def expand(self, pcoll):
...
+
+    class LogValueProvidersFn(beam.DoFn):
+      def __init__(self, string_vp):
+        self.string_vp = string_vp
+
+      # Define the DoFn that logs the ValueProvider value.
+      # The DoFn is called when creating the pipeline branch.
+      # This example logs the ValueProvider value, but
+      # you could store it by pushing it to an external database.
+      def process(self, an_int, **kwargs):
```
Comment (pabloem) [inline sdks/python/apache_beam/examples/snippets/snippets_test.py]: You don't need the `kwargs`.

```suggestion
      def process(self, an_int):
```

File: sdks/python/apache_beam/examples/snippets/snippets_test.py [hunk 3]
```diff
@@ -1266,6 +1266,59 @@ def expand(self, pcoll):
...
+    # Create pipeline.
+    with beam.Pipeline(options=my_options) as p:
+      # Add a branch for logging the ValueProvider value.
+      vp_output = (p
+                   | beam.Create([None])
+                   | 'LogValueProvider' >> beam.ParDo(
+                LogValueProvidersFn(my_options.string_value)))
+
+      # Test value provider argument : is equal to given string 'the quick brown fox jumps over the lazy dog'
+      assert_that(vp_output, equal_to(['the quick brown fox jumps over the lazy dog']),
```
Comment (pabloem) [inline sdks/python/apache_beam/examples/snippets/snippets_test.py]: If you move this to the end, you can end the snippet before the assertion
```
# [END AccessingValueProviderInfoAfterRunSnip1]

assert_that(main_pipeline_output, equal_to([6]), label='assert_main_output')
assert_that(vp_output, equal_to(['the quick brown fox jumps over the lazy dog']),
```

Comment (angulartist) [reply sdks/python/apache_beam/examples/snippets/snippets_test.py]: Thanks

File: sdks/python/apache_beam/examples/snippets/snippets.py (hunk 1 above)
Comment (angulartist) [reply sdks/python/apache_beam/examples/snippets/snippets.py]: Sure, I've moved it to snippets_test.py

**Comment by pabloem** [comment]
Thanks for adding the snippets. There's only one more issue, with lints:
```
01:02:12 > Task :sdks:python:test-suites:tox:py2:lintPy27
01:02:12 ************* Module apache_beam.examples.snippets.snippets_test
01:02:12 C:1281, 0: Line too long (97/80) (line-too-long)
01:02:12 C:1306, 0: Wrong hanging indentation (add 7 spaces).
01:02:12                 LogValueProvidersFn(my_options.string_value)))
01:02:12                 ^      | (bad-continuation)
01:02:12 C:1315, 0: Line too long (87/80) (line-too-long)
01:02:12 C:1317, 0: Line too long (82/80) (line-too-long)
01:02:12 C:1318, 0: Trailing whitespace (trailing-whitespace)
01:02:12 
01:02:12 --------------------------------------------------------------------
01:02:12 Your code has been rated at 10.00/10 (previous run: 10.00/10, -0.00)
```

**Comment by aaltay** [comment]
@angulartist are you still working on this PR?

**Comment by angulartist** [comment]
Hey yes I'm working on it

**Pull request closed without merging**